### PR TITLE
Processetti 2

### PR DESCRIPTION
**Pull request by JacopoCossu**
Ecco

### CODE REVIEW

File: pages/processetti/strumenti/bibliografia.tsx [hunk 1]
```diff
@@ -12,78 +12,77 @@ const Bibliografia: NextPage<BibliografiaProps> = (props) => {
...
           parte II, Roma, pp. 647-99.
         </p>
 
         <p>
-          Zannini Andrea (1998), ‘L’altra Bergamo in laguna: la comunità
-          bergamasca a Venezia’, in Storia economica e sociale di Bergamo. Il
-          tempo della Serenissima. Il lungo Cinquecento, Bergamo, Fondazione per
-          la storia economica e sociale di Bergamo, pp. 175-93.
+          Zannini Andrea (1998), &quote;L'altra Bergamo in laguna: la comunità
+          bergamasca a Venezia&quote;, in <i>Storia economica e sociale di Bergamo. Il
```
Comment (joschne) [inline pages/processetti/strumenti/bibliografia.tsx:84]: Replace ' by &apos;
```suggestion
          Zannini Andrea (1998), &quot;L&apos;altra Bergamo in laguna: la comunità
```

File: pages/processetti/strumenti/bibliografia.tsx [hunk 2]
```diff
@@ -12,78 +12,77 @@ const Bibliografia: NextPage<BibliografiaProps> = (props) => {
...
 
         <p>
-          Canepari Eleonora (2003), ‘Mestiere e spazio urbano nella costruzione
-          dei legami sociali degli immigrati a Roma in età moderna’, in A. Arru
-          e F. Ramella (a cura di), L’Italia delle migrazioni interne. Donne,
-          uomini, mobilità in età moderna e contemporanea, Roma, pp. 33-76.
+          Canepari Eleonora (2003), &quote;Mestiere e spazio urbano nella costruzione
+          dei legami sociali degli immigrati a Roma in età moderna&quote;, in A. Arru
+          e F. Ramella (a cura di), <i>L'Italia delle migrazioni interne. Donne,
+          uomini, mobilità in età moderna e contemporanea</i>, Roma, pp. 33-76.
```
Comment (joschne) [inline pages/processetti/strumenti/bibliografia.tsx:41]: Replace ' by &apos;

```suggestion
          e F. Ramella (a cura di), <i>L&apos;Italia delle migrazioni interne. Donne,
```

File: pages/processetti/strumenti/bibliografia.tsx [hunk 3]
```diff
@@ -12,78 +12,77 @@ const Bibliografia: NextPage<BibliografiaProps> = (props) => {
...
+          Seicento attraverso i processetti matrimoniali</i>, tesi di laurea, dir.
           M. A. Visceglia, La Sapienza Università di Roma.
         </p>
         <p>
-          Avorio Tiziana, Chianese Stefano, Guarino Nicola (2003), ‘Una città
+          Avorio Tiziana, Chianese Stefano, Guarino Nicola (2003), &quote;Una città
           senza immigranti? Caratteri della mobilità a Napoli tra Settecento e
-          Ottocento’, in A. Arru e F. Ramella (a cura di), L’Italia delle
+          Ottocento&quote;, in A. Arru e F. Ramella (a cura di), <i>L'Italia delle
           migrazioni interne. Donne, uomini, mobilità in età moderna e
```
Comment (joschne) [inline pages/processetti/strumenti/bibliografia.tsx:23]: Replace ' by &apos;

```suggestion
          Ottocento&quote;, in A. Arru e F. Ramella (a cura di), <i>L&apos;Italia delle
```

File: pages/processetti/strumenti/bibliografia.tsx
Comment (joschne) [inline pages/processetti/strumenti/bibliografia.tsx:1]: Replace `&quote;` with `&quot;` everywhere in this file.

Comment (JacopoCossu) [reply pages/processetti/strumenti/bibliografia.tsx:1]: oops, sorry

File: pages/processetti/metodologia/index.tsx [hunk 4]
```diff
@@ -1,16 +1,194 @@
...
+        <br />
+        <br />
+
+        <p>
+          Per una più analitica descrizione della metodologia e della semantica
+          sottesa alla definizione infrastrutturale di Geovistory si vedano:
+        </p>
+        <p>
+        <a name="2013">
+          Beretta Francesco, Butez Charlotte (2013), “Un SIG Collaboratif Pour
```
Comment (joschne) [inline pages/processetti/metodologia/index.tsx:154]: The `<a>`-tag is the link you click on. The `href=#2013` will target the element with `id=2013` on that page. I suggest this: 

```suggestion
        <p id="2013">
          Beretta Francesco, Butez Charlotte (2013), “Un SIG Collaboratif Pour
          La Recherche Historique”, in “Géomatique Expert”, 91, pp. 30-35 / 92,
          pp. 48-54
        </p>

        <p id="2016">
          Beretta Francesco (2016), “Pour une annotation sémantique des textes:
          le projet symogih.org et la Text encoding initiative”, in “Bruniana e
          Campanelliana, Ricerche filosofiche e materiali storico - testuali”,
          XXII.2 , pp. 453-65 [https://doi.org/10.19272/201604102005]
        </p>

        <p id="2021">
          Beretta Francesco (2021), “A Challenge for Historical Research: Making
          Data FAIR Using a Collaborative Ontology Management Environment
          (OntoME)”, in “Semantic Web”, 12.2, pp. 279-94
          [https://doi.org/10.3233/SW-200416]
        </p>

        <p id="2022">
          Beretta Francesco (2022), “Interopérabilité Des Données de La
          Recherche et Ontologies Fondationnelles: Un Écosystème d”extensions Du
          CIDOC CRM Pour Les Sciences Humaines et Sociales”, in N. Lasolle, O.
          Bruneau, J. Lieber (a cura di),{' '}
          <i>Actes Des Journées Humanités Numériques et Web Sémantique</i>,
          Nancy, pp. 2-22 [https://doi.org/10.5281/zenodo.7014341]
        </p>

        <p id="2023">
          Beretta Francesco (2023), “Données ouvertes liées et recherche
          historique: un changement de paradigme”, in “Humanités numériques”, 7
          [https://doi.org/10.4000/revuehn.3349]
        </p>
```

**Review by joschne — CHANGES_REQUESTED** [review]
I request some changes. See my comments.

**Review by joschne — CHANGES_REQUESTED** [review]
I request some changes. See my comments.

File: pages/processetti/metodologia/index.tsx [hunk 5]
```diff
@@ -1,16 +1,225 @@
...
+          <i>Actes Des Journées Humanités Numériques et Web Sémantique</i>,
+          Nancy, pp. 2-22 [https://doi.org/10.5281/zenodo.7014341]
+        </p>
+
+        <p id="2023">
+          Beretta Francesco (2023), “Données ouvertes liées et recherche
+          historique: un changement de paradigme”, in “Humanités numériques”, 7
+          [https://doi.org/10.4000/revuehn.3349]
+        </p>
+          La Recherche Historique”, in “Géomatique Expert”, 91, pp. 30-35 / 92,
```
Comment (joschne) [inline pages/processetti/metodologia/index.tsx:186]: Something is wrong here

File: pages/processetti/strumenti/bibliografia.tsx (hunk 1 above)
Comment (joschne) [reply pages/processetti/strumenti/bibliografia.tsx]: There is a closing `</i>` without opening `<i>``

**Pull request merged**